build/cli.js

```diff
@@ -19,12 +24,9 @@ const {
   HOMEPAGE_FEED_DISPLAY_MAX,
 } = require("./constants");
 const { makeSitemapXML, makeSitemapIndexXML } = require("./sitemaps");
-const {
-  CONTENT_TRANSLATED_ROOT,
-  CONTENT_ROOT,
-} = require("../content/constants");
 const { uniqifyTranslationsOf } = require("./translationsof");
 const { humanFileSize } = require("./utils");
+const { syncAllTranslatedContent } = require("./sync-translated-content");
```

Comment:
I think this should be:
```suggestion
const { syncTranslatedContentForAllLocales } = require("./sync-translated-content");
```

Reply:
Good catch. That was due to the grand renaming :/